mk/top_of_makefile.mk

```diff
@@ -231,7 +230,9 @@ CPPFLAGS += \
 # it's use seems questionable for that kind of target anyway.
 # The launchpad.net arm-none-eabi-gcc toolchain (at least) uses -fshort-enums.
 ifneq ($(TARGET_SYS),none)
-CPPFLAGS += -fstack-protector
+	ifneq ($(findstring redox,$(TARGET_SYS)),redox)
```

Comment:
Is there a Redox issue that we could link to in a comment? Or if not, could you create one and then link it?